### PR TITLE
Implemented entity operations update and functests

### PR DESCRIPTION
**Pull request by joaoGabriel55**

### CODE REVIEW

File: src/lib/orionld/serviceRoutines/orionldPostBatchUpdate.cpp [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    else
+      kjChildRemove(incomingTree, entityP);
+
+    entityP = next;
+  }
+
+  //
+  // 02. Check if some ID from idArray not exists
+  //
+  for (KjNode* idEntity = idArray->value.firstChildP; idEntity != NULL; idEntity = idEntity->next)
```
Comment (kzangeli) [inline src/lib/orionld/serviceRoutines/orionldPostBatchUpdate.cpp:155]: Don't we have a function to do this check with a single access to mongo?
If don't have one, we need to implement one.
Can't perform one DB access per entity ... way too slow

File: test/functionalTest/cases/0000_ngsild/ngsild_entity_post_operations_update_error_handling.test [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+{
+  "success": [
+    "urn:ngsi-ld:entity:E1"
+  ],
+  "errors": [
+    {
+      "entityId": "urn:ngsi-ld:entity:E3",
+      "error": {
+        "type": "https://uri.etsi.org/ngsi-ld/errors/BadRequestData",
+        "title": "entity not exists",
```
Comment (kzangeli) [inline test/functionalTest/cases/0000_ngsild/ngsild_entity_post_operations_update_error_handling.test]: Change text to "entity does not exist"

**Comment by kzangeli** [comment]
Good job!
Just a few details.
Well, one major one about avoiding extra calls to the DB.
You have an error or two in "Compliance Tests", those need to be addressed.

File: src/lib/orionld/serviceRoutines/orionldPostBatchUpdate.cpp (hunk 1 above)
Comment (joaoGabriel55) [reply src/lib/orionld/serviceRoutines/orionldPostBatchUpdate.cpp:155]: All right, Ken

**Pull request merged**